### ISSUE TITLE
2.4.0 Owntracks keeps crashing

### ISSUE THREAD

**Issue by matt-8319** [owntracks/android#1081]
Latest version of owntracks keeps crashing. Well not display contacts in friends list and randomly crashes when navigating between menu options.

```
FATAL EXCEPTION: main
Process: org.owntracks.android, PID: 15327
java.lang.RuntimeException: Unable to pause activity {org.owntracks.android/org.owntracks.android.ui.map.MapActivity}: kotlin.UninitializedPropertyAccessException: lateinit property callbackWrapper has not been initialized
	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:4743)
	at android.app.ActivityThread.performPauseActivity(ActivityThread.java:4694)
	at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:4645)
	at android.app.servertransaction.PauseActivityItem.execute(PauseActivityItem.java:46)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7664)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property callbackWrapper has not been initialized
	at org.owntracks.android.ui.map.MapLocationSource.deactivate(MapLocationSource.kt:52)
	at org.owntracks.android.ui.map.GoogleMapFragment.onPause(GoogleMapFragment.kt:188)
	at androidx.fragment.app.Fragment.performPause(Fragment.java:3168)
	at androidx.fragment.app.FragmentStateManager.pause(FragmentStateManager.java:632)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:314)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:112)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1647)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3128)
	at androidx.fragment.app.FragmentManager.dispatchPause(FragmentManager.java:3090)
	at androidx.fragment.app.FragmentController.dispatchPause(FragmentController.java:284)
	at androidx.fragment.app.FragmentActivity.onPause(FragmentActivity.java:390)
	at org.owntracks.android.ui.base.BaseActivity.onPause(BaseActivity.java:179)
	at android.app.Activity.performPause(Activity.java:8168)
	at android.app.Instrumentation.callActivityOnPause(Instrumentation.java:1510)
	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:4733)
	... 12 more
```
* **App build number**: 2.4.0
* **Android version**: CalyxOS 2.8.0 Android 11
* **Device**: pixel 4a

**Comment by growse** [comment]
I'll take a look, thanks.

**Comment by growse** [comment]
Erugh - this actually got fixed in 80b8d068c67b528d187ae317504670f6176a9881, but that's only on the 2.5 branch. I'll cherry-pick it and do a new release. Thanks!
